mslib/msui/wms_control.py

```diff
@@ -390,6 +390,7 @@ class WMSControlWidget(QtWidgets.QWidget, ui.Ui_WMSDockWidget):
     fetch = QtCore.pyqtSignal([list], name="fetch")
     signal_disable_cbs = QtCore.Signal(name="disable_cbs")
     signal_enable_cbs = QtCore.Signal(name="enable_cbs")
+    image_displayed = QtCore.pyqtSignal()
```

Comment:
I'm unsure, if I later know that this is meant as a signal. The others are all have a prefix.